templates/ethstore.html

```diff
@@ -0,0 +1,524 @@
+                </svg>
+              </div>
+              <div class="h5 text-muted feature-text-container" style="padding-top: 10px;">
+                <span class="feature-text"> ETH.STORE is transparent, fair and easily auditable </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="my-5">
+        <h1 class="title-head">ETH.STORE Explained</h1>
+
+        <button type="button" class="collapsible">What is ETH.STORE?</button>
+        <div class="content">
+          <p>ETH.STORE (Ether <strong>St</strong>aking <strong>O</strong>ffered <strong>R</strong>at<strong>e</strong>) is a transparent Ethereum staking reward reference rate. It represents the average financial return validators on the Ethereum network have achieved in a 24-hour period as published on https://staking.ethermine.org/statistics.</p>
+        </div>
+
+        <button type="button" class="collapsible">How are active validators counted by ETH.STORE?</button>
+        <div class="content">
+          <p>All validators that stayed in the active state during the entire reward day are counted. New validators joining or exiting mid day are excluded.</p>
+        </div>
+
+        <button type="button" class="collapsible">How long is a reward day and when does it start?</button>
+        <div class="content">
+          <p>Each reward day starts at 12:00:23 UTC and lasts for 24 hours (or exactly 225 beaconchain epochs).</p>
+        </div>
+
+        <button type="button" class="collapsible">How are total rewards calculated in ETH.STORE?</button>
+        <div class="content">
+          <p>Total staking rewards consists of consensus rewards and transaction fees:</p>
+          <ol>
+            <li>Consensus rewards: Rewards received by validators for proposing and attesting in accordance with the consensus rules of the Ethereum protocol</li>
+            <li>Transaction fees: Fees received by validators for processing transactions on the Ethereum network</li>
+          </ol>
+          <p>Total staking rewards equals the sum of consensus rewards and transaction fees</p>
+        </div>
+
+        <button type="button" class="collapsible">How can I reproduce the ETH.STORE calculation on my own?</button>
+        <div class="content">
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>Variable</th>
+                  <th>Description</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Start epoch (SE):</td>
+                  <td>First epoch after 1200 UTC every day</td>
```

Comment:
Above the notation has been "12:00:23 UTC" but here the ":" is missing in "1200 UTC". Shouldn't this be harmonized?

Again, this matches the behavior over on https://staking.ethermine.org/statistics but I think this should be changed on both sides.

Reply:
Fixed in new commit